docker/agent/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG AUTONOMY_IMAGE_VERSION="latest"
```

Comment:
I don't like how we now have code duplication again in each repo. We need to do this differently.